gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -743,6 +743,49 @@ protected GoogleCloudStorageItemInfo getInitialMetadata() {
     };
   }
 
+  public SeekableByteChannel open(
```

Comment:
Should we add this to the interface as well (with a default impl) ?

Reply:
I think we do. Can @medb confirm?

Reply:
Yes, sgtm